test/src/main/java/org/axonframework/test/saga/FixtureConfiguration.java

```diff
@@ -126,6 +126,15 @@
      */
     FixtureConfiguration registerHandlerDefinition(HandlerDefinition handlerDefinition);
 
+    /**
+     * Registers a callback to be invoked when the fixture execution starts recording. This happens right before
+     * invocation of the 'when' step (stimulus) of the fixture.
```

Comment:
What would you think of adding a short "and this is what you could use this for" in this javadoc? Granted, the Reference Guide should discuss this of course, but a one sentence `e.g.` wouldn't be bad I'd assume. What's your take on that @robvdlv?